app/controllers/users/phones_controller.rb

```diff
@@ -32,6 +32,7 @@ def update
       @presenter = PhoneSetupPresenter.new(delivery_preference)
       if @user_phone_form.submit(user_params).success?
         process_updates
+        set_default_phone
```

Comment:
Is there a reason you are doing this in the controller instead of in the phone form?

Reply:
The user_phone_form currently updates the user model using the UpdateUser service. Until now, all fields on the form were stored on the user model. Then we trigger an update to phone_configurations. 

Updating made_default_at in the controller seemed like less of a hack (To me) then adding made_default_at to the UpdateUser service. I know we plan on removing the phone_configurations fields from the user model.

Reply:
I spent some time yesterday digging into the issue. I will remove set_default_phone from this controller in the next update.